docs/reference/data-frames/apis/put-transform.asciidoc

```diff
@@ -67,6 +67,15 @@ IMPORTANT:  You must use {kib} or this API to create a {dataframe-transform}.
 `source`::
   (object) Required. The source configuration, which consists of `index` and 
   optionally a `query`. See <<data-frame-transform-source>>.
+  
+`sync`::
+  (Optional, object) Defines the properties required to run continuously.
+  `field`:::
+    (Required, string) The date field that is used to identify new documents in
+    the source.
+  `delay`:::
+    (Optional, time units) The time delay between the current time and the
```

Comment:
we identified yesterday that we need a default value for delay, I will update this comment once we agreed on a value.